dss/operations/checkout.py

```diff
@@ -0,0 +1,191 @@
+"""
+Tools for managing checkout buckets
+"""
+import typing
+import argparse
+import logging
+import json
+import time
+import collections
+
+from dss import Replica
+from dss.operations import dispatch
+from dss import Config
+from cloud_blobstore import BlobNotFoundError
+from dss.api.bundles import get_bundle_manifest
+from dss.storage.hcablobstore import compose_blob_key
+from dss.storage.identifiers import DSS_BUNDLE_KEY_REGEX, VERSION_REGEX, UUID_REGEX, FILE_PREFIX
+from dss.storage.checkout.bundle import verify_checkout as bundle_checkout
+from dss.api.files import _verify_checkout as file_checkout
+from dss.storage.checkout import cache_flow
+
+logger = logging.getLogger(__name__)
+
+
+class CheckoutHandler:
+    def __init__(self, argv: typing.List[str], args: argparse.Namespace):
+        if args.keys is not None:
+            self.keys = args.keys.copy()
+        else:
+            self.keys = None
+        self.replica = Replica[args.replica]
+        self.handle = Config.get_blobstore_handle(self.replica)
+        self.bucket = self.replica.checkout_bucket
+
+    def process_command_locally(self, argv: typing.List[str], args: argparse.Namespace):
```

Comment:
It looks like this method can be removed and its contents moved into `__call__`.